pkg/kubectl/cmd/resize.go

```diff
@@ -94,6 +94,9 @@ func RunResize(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []str
 	if err != nil {
 		return err
 	}
+	if len(infos) > 1 {
+		return fmt.Errorf("Multiple resources provided: %v", args)
```

Comment:
Probably needs to match other commands that don't allow multiple.  Also should be in standard form (lowercase first letter)